src/abi/tests.rs

```diff
@@ -104,10 +109,10 @@ fn constants_and_types() {
         }
     );
 
-    assert_eq!(idl.types[1].name, "Week");
-    assert!(idl.types[1].docs.is_none());
+    assert_eq!(idl.types[0].name, "Week");
+    assert!(idl.types[0].docs.is_none());
     assert_eq!(
-        idl.types[1].ty,
+        idl.types[0].ty,
```

Comment:
Would you mind placing the check for item `0` before that for item `1`? I believe it improves readability.